.github/workflows/nodejs.yml

```diff
@@ -48,4 +47,33 @@ jobs:
         sudo ifconfig docker0 10.200.10.1/24
         ${GITHUB_WORKSPACE}/streamr-docker-dev/streamr-docker-dev/bin.sh start --wait
         npm run test-integration
-
+  streamr-client-testing-tool:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+    - uses: actions/checkout@v1
+    - name: streamr-client tool test
+      continue-on-error: true
+      run: |
+        sudo service mysql stop
+        if [ ! -d streamr-docker-dev ]; then # Skip clone on subsequent attemps.
+         git clone https://github.com/streamr-dev/streamr-docker-dev.git
+        fi
+        sudo ifconfig docker0 10.200.10.1/24
+        ${GITHUB_WORKSPACE}/streamr-docker-dev/streamr-docker-dev/bin.sh start smart-contracts-init nginx engine-and-editor --wait
+        echo "Testing Tool setup"
+        git clone https://github.com/streamr-dev/streamr-client-testing.git
+        cd ${GITHUB_WORKSPACE}/streamr-client-testing
+        echo "Setup packagejson to use PR JS client"
+        sed -i "s/com.streamr:client:1.3.0/com.streamr:client:+/g" build.gradle
+        sed -i "s/\"streamr-client\": \"\^3.1.2\"/\"streamr-client\":\"PATH\"/g" package.json
```

Comment:
it should work only with version `^3.1.2`?

Reply:
The testing client made by @mthambipillai has that value there. I don't know if it's essential or not. What is actually being gone there (if you check a bit lower) is changing the version to point to the local code.